### PR TITLE
ITSASU-1279: enable overseas property cnotinue button

### PR DESCRIPTION
**Pull request by scullythornton1**

### CODE REVIEW

File: app/controllers/agent/business/OverseasPropertyCheckYourAnswersController.scala [hunk 1]
```diff
@@ -61,9 +61,13 @@ class OverseasPropertyCheckYourAnswersController @Inject()(val overseasPropertyC
       withAgentReference { reference =>
         if (isEnabled(SaveAndRetrieve)) {
           withOverseasProperty(reference) { property =>
-            subscriptionDetailsService.saveOverseasProperty(reference, property.copy(confirmed = true)).map(_ => {
-              Redirect(controllers.agent.routes.TaskListController.show())
-            })
+            if (property.accountingMethod.isDefined && property.startDate.isDefined) {
+              subscriptionDetailsService.saveOverseasProperty(reference, property.copy(confirmed = true)).map(_ => {
+                Redirect(controllers.agent.routes.TaskListController.show())
+              })
```
Comment (AlexRimmerHMRC) [inline app/controllers/agent/business/OverseasPropertyCheckYourAnswersController.scala:67]:
```suggestion
              subscriptionDetailsService.saveOverseasProperty(reference, property.copy(confirmed = true)).map { _ =>
                Redirect(controllers.agent.routes.TaskListController.show())
              }
```

File: app/views/agent/business/OverseasPropertyCheckYourAnswers.scala.html [hunk 2]
```diff
@@ -52,7 +52,7 @@
         caption = messages("agent.business.check-your-answers.content.overseas-property.caption"),
         isIndividual = false,
         postAction = postAction,
-        isComplete = viewModel.startDate.isDefined && viewModel.accountingMethod.isDefined,
+        isComplete = true,
```
Comment (AlexRimmerHMRC) [inline app/views/agent/business/OverseasPropertyCheckYourAnswers.scala.html:55]: As mentioned on the PR for UKProperty, this field is no longer necessary as we should never be disabling buttons.
It should be removed as a parameter to the check your answers helper

File: app/views/individual/incometax/business/OverseasPropertyCheckYourAnswers.scala.html [hunk 3]
```diff
@@ -52,7 +52,7 @@
         caption = messages("business.check-your-answers.content.overseas-property.caption"),
         isIndividual = true,
         postAction = postAction,
-        isComplete = viewModel.startDate.isDefined && viewModel.accountingMethod.isDefined,
+        isComplete = true,
```
Comment (AlexRimmerHMRC) [inline app/views/individual/incometax/business/OverseasPropertyCheckYourAnswers.scala.html:55]: As mentioned on the PR for UKProperty, this field is no longer necessary as we should never be disabling buttons.
It should be removed as a parameter to the check your answers helper

File: test/views/agent/business/OverseasPropertyCheckYourAnswersViewSpec.scala [hunk 4]
```diff
@@ -124,10 +125,10 @@ class OverseasPropertyCheckYourAnswersViewSpec extends ViewSpec {
...
         }
 
-        "have a continue button if confirmed" in {
-          val buttonLink: Element = document(viewModel = confirmedAccrualsProperty).selectHead(".govuk-button")
-          buttonLink.text mustBe OverseasPropertyCheckYourAnswers.continue
-        }
+//        "have a continue button if confirmed" in {
+//          val buttonLink: Element = document(viewModel = confirmedAccrualsProperty).selectHead(".govuk-button")
+//          buttonLink.text mustBe OverseasPropertyCheckYourAnswers.continue
+//        }
```
Comment (AlexRimmerHMRC) [inline test/views/agent/business/OverseasPropertyCheckYourAnswersViewSpec.scala:131]:
```suggestion

```

File: app/controllers/agent/business/OverseasPropertyCheckYourAnswersController.scala (hunk 1 above)
Comment (scullythornton1) [reply app/controllers/agent/business/OverseasPropertyCheckYourAnswersController.scala]: done

File: app/views/agent/business/OverseasPropertyCheckYourAnswers.scala.html (hunk 2 above)
Comment (scullythornton1) [reply app/views/agent/business/OverseasPropertyCheckYourAnswers.scala.html]: done

File: app/views/individual/incometax/business/OverseasPropertyCheckYourAnswers.scala.html (hunk 3 above)
Comment (scullythornton1) [reply app/views/individual/incometax/business/OverseasPropertyCheckYourAnswers.scala.html]: done

File: test/views/agent/business/OverseasPropertyCheckYourAnswersViewSpec.scala (hunk 4 above)
Comment (scullythornton1) [reply test/views/agent/business/OverseasPropertyCheckYourAnswersViewSpec.scala]: doe

**Pull request merged**